cfgov/unprocessed/apps/youth-employment-success/css/main.less

```diff
@@ -309,6 +314,12 @@ input:-moz-ui-invalid {
     background: transparent;
     border: none;
     padding: 0;
+
+    // Font size is being calculated at 13.875px, when the CFPBs guidelines state that
```

Comment:
Interesting. Is the em calculation off due to rounding I wonder? We may want ems here though. Should the user be able to change this font size?

Reply:
I guess so? I'm not really sure how to proceed then, we want the text to be 14px as a base, and I don't know how else to get it to be that size without the style getting overwritten

Reply:
Maybe I'm misunderstanding, but I'm using `px` because the `em`s seem to be miscalculated.

Reply:
If I set this to `unit( 14 / @base-font-size-px, em )` the computed font size is `14px`, in fact, if I remove this entirely the computed font size is still 14px. Where are you seeing `13.875px`?

<img width="521" alt="Screen Shot 2019-10-23 at 5 10 56 PM" src="https://user-images.githubusercontent.com/704760/67434384-1cf33800-f5b8-11e9-87a4-4ed527608787.png">

Reply:
Interesting, for both the designer and I, on chrome and firefox, we see the `.875em` value for `small` tags as `13.875`

Screenshot from chrome
<img width="485" alt="Screen Shot 2019-10-23 at 3 14 22 PM" src="https://user-images.githubusercontent.com/1421848/67438203-b49c5a80-f5a7-11e9-8d50-ba7cf587d138.png">

Reply:
I'll just take the rule off then, since it only seems to be affecting us.